cmd/snap/cmd_get.go

```diff
@@ -326,6 +340,22 @@ func (x *cmdGet) getConfdb(confdbViewID string, confKeys []string) (map[string]a
 	return conf, nil
 }
 
+func buildDefaultOutput(request string, defaultRaw string) (map[string]any, error) {
+	var defaultVal any
+	if err := json.Unmarshal([]byte(defaultRaw), &defaultVal); err != nil {
+		var merr *json.SyntaxError
+		if !errors.As(err, &merr) {
+			// shouldn't happen as we treat unmarshable values as strings
+			return nil, fmt.Errorf("internal error: cannot unmarshal --default value: %v", err)
+		}
+
```

Comment:
I think this should be an error if -t was provided

Reply:
done